src/components/LightGallery.astro

```diff
@@ -12,25 +12,22 @@ type Props = AstroLightGalleryType
 const {
   options = {},     // https://www.lightgalleryjs.com/docs/settings/
   addPlugins = [],
-  id = 'astro-lightgallery-' + Math.random().toString(36).slice(2, 11),
-  class: className = '',
+  id = `astro-lightgallery-${Math.random().toString(36).slice(2, 11)}`,
+  class: className = null,
   layout,
   ...props
 } = Astro.props;
 
 const paramLayout = layout ? (({ imgs, ...o }) => o)(layout) : {}    // remove imgs from layout
-const classContainer = (layout && layout.classContainer) ? layout.classContainer : ''
-const classItem = (layout && layout.classItem) ? layout.classItem : ''
+
+const classContainer = layout?.classContainer ?? null
+const classItem = layout?.classItem ??  null
 
 // update missing informations (if any) in layout.imgs
 if (layout) {
   layout.imgs.forEach(img => {
-    if (!img.srcThumb) {
-      img.srcThumb = img.src
-    }
-    if (!img.alt) {
-      img.alt = ''
-    }
+    if (!img.srcThumb) img.srcThumb = img.src

```

Comment:
No, I prefer the bracket version which is more readable according to me

Reply:
Changed :)